spec.html

```diff
@@ -18286,7 +18289,7 @@ <h1>
           <dd>It determines if its argument is a valid regular expression literal.</dd>
         </dl>
         <emu-alg>
-          1. If FlagText of _literal_ contains any code points other than `d`, `g`, `i`, `m`, `s`, `u`, or `y`, or if it contains the same code point more than once, return *false*.
+          1. If FlagText of _literal_ contains any code points other than `d`, `g`, `i`, `m`, `s`, `u`, or `y`, or FlagText of _literal_ contains any code point more than once, return *false*.
           1. Let _patternText_ be BodyText of _literal_.
           1. If FlagText of _literal_ contains `u`, let _u_ be *true*; else let _u_ be *false*.
```

Comment:
Readability suggestion:
```suggestion
          1. Let _flags_ be FlagText of _literal_.
          1. If _flags_ contains any code points other than `d`, `g`, `i`, `m`, `s`, `u`, or `y`, or _flags_ contains any code point more than once, return *false*.
          1. If _flags_ contains `u`, let _u_ be *true*; else let _u_ be *false*.
          1. Let _patternText_ be BodyText of _literal_.
```